java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -48,4 +48,29 @@ public interface StringCommands {
      *     is set, return the old value as a <code>String</code>.
      */
     CompletableFuture<String> set(String key, String value, SetOptions options);
+
+    /**
+     * Decrement the number stored at <code>key</code> by one. If <code>key</code> does not exist, it
+     * is set to 0 before performing the operation.
+     *
+     * @see <a href="https://redis.io/commands/decr/">redis.io</a> for details.
+     * @param key The key to decrement its value.
+     * @return The value of <code>key</code> after the decrement. An error is raised if <code>key
```

Comment:
please align the documentation with node/python, same in all places
(remove message ab error)